feat: add step to perform image data migration

jackhodgkiss opened the pull request:
The command `pulpcore-manager container-handle-image-data`[0] can be used to resolve issues with missing image fields.

[0]: https://github.com/pulp/pulp_container/pull/1554

jackhodgkiss committed:
feat: add step to perform image data migration

The command `pulpcore-manager container-handle-image-data`[0] can be used
to resolve issues with missing image fields. This will happen post
deployment of Pulp.

[0]: https://github.com/pulp/pulp_container/pull/1554

mnasiadka commented:
Why not add that to etc/kayobe/containers/pulp/post.yml? I understand that it won't break anything?

jackhodgkiss commented:
> Why not add that to etc/kayobe/containers/pulp/post.yml? I understand that it won't break anything?

Updated docs and added task to `post.yml` and tested all works.
